docs/api-docs/channels/channel-webhooks.md

```diff
@@ -0,0 +1,85 @@
+# Channel Webhooks
+
+<div class="otp" id="no-index">
+
+### On This Page
+- [Channel webhook events](#channel-webhook-events)
+- [Channel webhook callbacks](#channel-webhook-callbacks)
+- [Creating a channel webhook](#creating-a-channel-webhook)
+- [Resources](#resources)
+
+</div>
+
+Developers building third-party sales channels or multi-storefront capabilities may need to be notified when changes are made to a channel in BigCommerce so that they can perform downstream actions required to setup or deactivate the channel. To facilitate this, we've added `store/channel` webook events that fire when a channel is created or updated. We'll assume you're faimilar with webhooks and briefly introduce the channel events in this article; however, if you would like an introduction to webhooks, see [Webhooks Overview](https://developer.bigcommerce.com/api-docs/getting-started/webhooks/about-webhooks).

```

Comment:
Few suggestions on this paragraph:

- It would be downstream actions required to setup, **manage**, or deactivate a channel (for example, if the merchant changes the name)
- Let's use some stronger language to indicate that using these webhooks is best practice in the following scenarios / use cases: 1) When the status of a headless storefront channel is updated (type = storefront, platform != bigcommerce) so that the headless storefront can be properly displayed or not to shoppers (i.e. if it's moved to inactive, storefront should be down and not visible to shoppers), 2) When the status of any storefront channel is updated so that app partners who rely on whether the storefront is live to shoppers or not can react in time (@bookernath do you know if we send any webhooks today when the store's DFM status changes (i.e. store is launched or put into down for mainteance)? I looked on dev docs and didn't see anything, so think this may be really valuable to app partners in SSF world as well. 3) When a marketplace, POS, or marketing sales channel is disconnected, so that products will stop being synced to the sales channel platform 

In fact that first use case makes me think we should include a mention on the headless developer documentation about using Channel webhooks as well. 

cc: @becomevocal @kristinapototska Are there any other use cases / best practices that you think should be called out when using Channel webhooks? (Any that are MSF specific should go in the MSF guide rather than here though, just FYI @aglensmith )